distributor.php

```diff
@@ -18,7 +18,7 @@
 	exit; // Exit if accessed directly.
 }
 
-define( 'DT_VERSION', '1.6.6-dev' );
+define( 'DT_VERSION', '1.6.7' );
```

Comment:
```suggestion
define( 'DT_VERSION', '1.6.7-dev' );
```

The `-dev` stays, build will strip it out :) Will clarify this in code and documentation separately from this release.